server/service/hosts.go

```diff
@@ -484,6 +484,34 @@ func (svc *Service) checkWriteForHostIDs(ctx context.Context, ids []uint) error
 	return nil
 }
 
+// //////////////////////////////////////////////////////////////////////////////
+// Get Host Lite
+// //////////////////////////////////////////////////////////////////////////////
+func (svc *Service) GetHostLite(ctx context.Context, id uint) (*fleet.Host, error) {
+	alreadyAuthd := svc.authz.IsAuthenticatedWith(ctx, authzctx.AuthnDeviceToken)
```

Comment:
AFAICS, no device (Fleet Desktop) endpoint uses this, so there's no need to add this thing.

Reply:
TIL.  it's removed